packages/website/src/content/03-01-touchpoint-components.md

```diff
@@ -118,6 +118,8 @@ const touchpoint = await create(touchpointOptions);
 
 ### Example CustomCard Modality
 
+<img src="/images/Touchpoint-Custom-Card.png" alt="Product Card with CustomCard" style="max-width: 40%;">
```

Comment:
Why max-width: 40% on mobile, isn't that a bit narrow? Should we consider doing image styling set for all markdown documents uniformly so you can just use the `![alt-text](url)` syntax?